docker/qgisserver/geomapfish_qgisserver/accesscontrol.py

```diff
@@ -226,19 +226,24 @@ def get_ogcserver_accesscontrol_config(self) -> None:
 
     def get_ogcserver_accesscontrol(self) -> "OGCServerAccessControl":
         parameters = self.serverInterface().requestHandler().parameterMap()
+
         if self.single:
             if "MAP" in parameters:
                 raise GMFException("The map parameter should not be provided")
             return self.ogcserver_accesscontrol
-        if "MAP" not in parameters:
-            raise GMFException("The map parameter should be provided")
-        if parameters["MAP"] not in self.ogcserver_accesscontrols:
+
+        if "MAP" in parameters:
+            config_uri = parameters["MAP"]
+        else:
+            config_uri = self.serverInterface().configFilePath()
```

Comment:
What does `self.serverInterface().configFilePath()`?
Especially when we on the landing page?

Reply:
When we are on the landing page, browser retrieve index.json and try GetMap with https://localhost:8484/mapservproxy/qgis/project/<hash>
And it that case we do not have any MAP parameter, but `self.serverInterface().configFilePath()` contains the project path.